.cloud_build/sketch_pad.yaml

```diff
@@ -3,5 +3,5 @@ steps:
     args: ['build', 'web']
     dir: pkgs/sketch_pad
   - name: gcr.io/$PROJECT_ID/firebase
-    args: ['deploy', '--project=$PROJECT_ID', '--only=hosting']
+    args: ['deploy', '--project=$PROJECT_ID', '--only=preview']
```

Comment:
Same here.

> Isn't the `--only` option the type of deploy you're doing and should still be `hosting`?

Reply:
You're right this should be "firebase deploy --only hosting:TARGET_NAME" as per https://firebase.google.com/docs/hosting/multisites